CHANGELOG.md

```diff
@@ -164,6 +164,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 * (types) [\#9511](https://github.com/cosmos/cosmos-sdk/pull/9511) Change `maxBitLen` of `sdk.Int` and `sdk.Dec`  to handle max ERC20 value.
 * (keyring) [\#9562](https://github.com/cosmos/cosmos-sdk/pull/9563) fix keyring kwallet backend when using with empty wallet.
 * (keyring) [\#9583](https://github.com/cosmos/cosmos-sdk/pull/9583) Fix correct population of legacy `Vote.Option` field for votes with 1 VoteOption of weight 1.
+* (x/genutil) [\#9574](https://github.com/cosmos/cosmos-sdk/pull/9575) Actually use the `gentx` client tx flags (like `--keyring-dir`)
```

Comment:
There was a CHANGELOG refactor in the meantime. @michaelfig  Could you move this to a new Bug Fixes section in Unreleased?

Reply:
> @michaelfig Could you move this to a new Bug Fixes section in Unreleased? (sorry about that)

No problem, done!